### DISCUSSION TITLE
Feature Request: Force Update Branch With Rebase

### DISCUSSION THREAD

**Discussion by aqeelat** [community/community#40347] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

Currently, when a PR is out of sync, the "Update branch" button shows up with two options: merge or rebase.
It would be awesome if the repo admin can disable the "Update with merge commit" option or at least make "Update with rebase" the default option.

**Comment by Loulouka24** [comment]
Switch to the branch/PR with your changes. Locally set your Git repo to the branch that has the changes you want merged in the target branch .
Execute the Git rebase command . 
Fix all and any conflicts . 
Force push the new history [.][(url)](https://louloukaformula.com/)

**Comment by gamefowl** [comment]
Switch to the branch/PR with your changes. Locally set your Git repo to the branch that has the changes you want merged in the target branch.
Execute the Git rebase command. ...
Fix all and any conflicts. ...
Force push the new history[.]([url](https://gamefowlsforsale.com/))

**Comment by bewuethr** [comment, marked as answer]
Related:  #12032
